docs/source/cite.rst

```diff
@@ -0,0 +1,8 @@
+.. _cite:
+
+How to cite T3
+==============
+
+Text form::
+
+    A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3
```

Comment:
Can we add some reasonable line breaks for better readability?

Reply:
Added some line breaks!